kuma/contributions/apps.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class ContributionsConfig(AppConfig):
+    name = 'kuma.contributions'
```

Comment:
It may make sense to add an "enabled" function here to consolidate the ``MDN_CONTRIBUTIONS`` check, so that the waffle flag is just here as well.

Reply:
Added ``enabled()`` in ``kuma/contributions/utils.py`` instead.